extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/extensions.html

```diff
@@ -1,4 +1,4 @@
-<section id="main" class="white-bg extensions header-tabs">
+<section id="main" class="white-bg extensions header-tabs" droppable>
```

Comment:
the attribute droppable is a directive defined in `extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.extension.js`. The directive adds drag-n-drop support to the whole `<section>` element.